addons/sale/wizard/__init__.py

```diff
@@ -7,3 +7,4 @@
 from . import payment_link_wizard
 from . import sale_make_invoice_advance
 from . import sale_order_cancel
+from . import mass_cancel_orders_wizard
```

Comment:
```suggestion
from . import mass_cancel_orders
```
we are in the wizard folder, no need to add the 'wizard' postfix everywhere.